# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../squashfs-tools/squashfs-tools-4.1-Makefile.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- squashfs4.1/squashfs-tools/Makefile.orig	2010-09-30 22:36:25.396000820 +0200
+++ squashfs4.1/squashfs-tools/Makefile	2010-09-30 23:17:39.896000821 +0200
@@ -91,7 +91,7 @@ CFLAGS = $(INCLUDEDIR) -D_FILE_OFFSET_BI
 	-D_GNU_SOURCE -DCOMP_DEFAULT=\"$(COMP_DEFAULT)\"  -O2 -Wall
 
 LIBS =
-ifdef GZIP_SUPPORT
+ifeq ($(GZIP_SUPPORT),1)
 CFLAGS += -DGZIP_SUPPORT
 MKSQUASHFS_OBJS += gzip_wrapper.o
 UNSQUASHFS_OBJS += gzip_wrapper.o
@@ -99,7 +99,7 @@ LIBS += -lz
 COMPRESSORS += gzip
 endif
 
-ifdef LZMA_SUPPORT
+ifeq ($(LZMA_SUPPORT),1)
 LZMA_OBJS = $(LZMA_DIR)/C/Alloc.o $(LZMA_DIR)/C/LzFind.o \
 	$(LZMA_DIR)/C/LzmaDec.o $(LZMA_DIR)/C/LzmaEnc.o $(LZMA_DIR)/C/LzmaLib.o
 INCLUDEDIR += -I$(LZMA_DIR)/C
@@ -109,7 +109,7 @@ UNSQUASHFS_OBJS += lzma_wrapper.o $(LZMA
 COMPRESSORS += lzma
 endif
 
-ifdef XZ_SUPPORT
+ifeq ($(XZ_SUPPORT),1)
 CFLAGS += -DLZMA_SUPPORT
 MKSQUASHFS_OBJS += xz_wrapper.o
 UNSQUASHFS_OBJS += xz_wrapper.o
@@ -117,7 +117,7 @@ LIBS += -llzma
 COMPRESSORS += lzma
 endif
 
-ifdef LZO_SUPPORT
+ifeq ($(LZO_SUPPORT),1)
 CFLAGS += -DLZO_SUPPORT
 ifdef LZO_DIR
 INCLUDEDIR += -I$(LZO_DIR)/include
@@ -129,8 +129,8 @@ LIBS += $(LZO_LIBDIR) -llzo2
 COMPRESSORS += lzo
 endif
 
-ifdef XATTR_SUPPORT
-ifdef XATTR_DEFAULT
+ifeq ($(XATTR_SUPPORT),1)
+ifeq ($(XATTR_DEFAULT),1)
 CFLAGS += -DXATTR_SUPPORT -DXATTR_DEFAULT
 else
 CFLAGS += -DXATTR_SUPPORT
@@ -142,7 +142,7 @@ endif
 #
 # If LZMA_SUPPORT is specified then LZO_DIR must be specified too
 #
-ifdef LZMA_SUPPORT
+ifeq ($(LZMA_SUPPORT),1)
 ifndef LZMA_DIR
 $(error "LZMA_SUPPORT requires LZMA_DIR to be also defined")
 endif
@@ -151,8 +151,8 @@ endif
 #
 # Both XZ_SUPPORT and LZMA_SUPPORT cannot be specified
 #
-ifdef XZ_SUPPORT
-ifdef LZMA_SUPPORT
+ifeq ($(XZ_SUPPORT),1)
+ifeq ($(LZMA_SUPPORT),1)
 $(error "Both XZ_SUPPORT and LZMA_SUPPORT cannot be specified")
 endif
 endif
